types/json/binary.go

```diff
@@ -192,6 +201,34 @@ func (bj BinaryJSON) GetString() []byte {
 	return bj.Value[lenLen : lenLen+int(strLen)]
 }
 
+// Opaque represents a raw binary type
+type Opaque struct {
+	// TypeCode is the same with database type code
+	TypeCode byte
+	// Buf is the underlying bytes of the data
+	Buf []byte
+}
+
+// GetOpaque gets the opaque value
+func (bj BinaryJSON) GetOpaque() Opaque {
+	typ := bj.Value[0]
```

Comment:
in the comment, the opaque data is `opaque ::= data-length typeId byte*`, but here seems it is `opaque ::= typeId data-length byte*`

Reply:
Nice catch. I reordered the layout for the convenience to get the type, but forgot to modify the comments :crying_cat_face: